docs/en/cloud/security/ip-egress-traffic-list.md

```diff
@@ -0,0 +1,50 @@
+---
+slug: /en/manage/security/ip-egress-traffic-list
+sidebar_label: IP Egress Traffic List
+title: IP Egress Traffic List
+---
+
+## Integrations could require egress access
+
+If you are using integration like the MySQL or PostgreSQL Engine, It's possible that you need to authorize ClickHouse Cloud to access to your instances.
+
+You could use this list of public IPs in `firewalls` or `Authorized networks` in GCP or in `Security Groups` in AWS or in any other infrastrucutre egrees management system you are using.
```

Comment:
Should we add a comment for Azure?

Reply:
I have no experience on Azure, let me see what can I find on the net !